src/Microsoft.AspNetCore.Dispatcher/AmbiguousEndpointException.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.AspNetCore.Dispatcher
+{
+    /// <summary>
+    /// An exception which indicates multiple matches in endpoint selection.
+    /// </summary>
+    [Serializable]
+    public class AmbiguousEndpointException : InvalidOperationException
```

Comment:
Let's not subclass `InvalidOperationException`, it's a little wierd. yeah the old code might have done this but we don't need it.